pkg/apis/eksctl.io/v1alpha5/validation.go

```diff
@@ -282,7 +282,7 @@ func validateNodeGroupBase(np NodePool, path string) error {
 		}
 	}
 
-	if instanceutils.IsGPUInstanceType(SelectInstanceType(np)) && ng.AMIFamily != NodeImageFamilyAmazonLinux2 {
+	if instanceutils.IsGPUInstanceType(SelectInstanceType(np)) && (ng.AMIFamily != NodeImageFamilyAmazonLinux2 && ng.AMIFamily != "") {
```

Comment:
can we add a test for condition checking `ng.AMIFamily != ""` throws expected error ?

Reply:
+1

Reply:
>can we add a test for condition checking ng.AMIFamily != "" throws expected error ?

There are several test cases in `pkg/apis/eksctl.io/v1alpha5/gpu_validation_test.go` that address the `ng.AMIFamily != ""` case.